src/effects-and-handlers.md

```diff
@@ -0,0 +1,102 @@
+# Effects and Handlers
+
+> **Warning:** Effects and handlers are an experimental feature. Do not use them
+> in production.
+
+> **Warning:** Do not use effects and handlers inside spawn expressions.
```

Comment:
Maybe inside is a bit confusing. `spawn(try { do Coin.flip(..) } with ...)` since its contained in the new spawn. Maybe `Do not use effects and handlers across threads.`

Reply:
You mean that handlers work correctly on other threads?

But the problem is that `spawn e` is not yet typed correctly.

Reply:
The type of the given expression above is pure so there should be no problem for typing? I just mean that `inside spawn expressions` seems unnecessarily specific, shouldn't the message just be "Don't mix spawn with control effects"

Reply:
I mean the type rule of spawn is broken.